src/labels/CustomLabels.labels

```diff
@@ -4880,6 +4880,136 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+        <value>Save Batch Number Format</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsDescDisplayFormat</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Explanation of Display Format options</shortDescription>
+        <value>Display Format allows you to control the appearance of your auto-numbered
+            field. It must be unique and include the number of digits as well as a prefix, if
+            desired. Your format can be up to 30 characters.&lt;/br&gt;&lt;/br&gt;
+            Examples&lt;/br&gt;
+            &lt;ul&gt;
+            &lt;li&gt;
+            A sequence formatted as {0} with a starting number of 9 displays as 9, 10, 11,
+            etc.
+            &lt;/li&gt;
+            &lt;li&gt;
+            A sequence formatted as {00000} with a starting number of 9 displays as 00009,
+            00010, 00011, etc.&#10;
+            &lt;/li&gt;
+            &lt;li&gt;
+            A sequence formatted as ABC-{0000} with a starting number of 9 displays as
+            ABC-0009, ABC-0010, ABC-0011, etc.
+            &lt;/li&gt;
+            &lt;/ul&gt;
+        </value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsError</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error message used when there is an error creating a Batch Number Format.</shortDescription>
+        <value>There was an error creating the Batch Number Format. Please review and try again. </value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsHeaderDisplayFormat</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for explanation of Display Format options</shortDescription>
+        <value>Learn more about display formats.</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsDescriptionCreate</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for creating a batch number format</shortDescription>
+        <value>Create a custom batch number format and choose a starting number. You must activate the new batch number format in the Batch Number Formats section.</value>
+    </labels>
+    <labels>
+        <fullName>autoNumberErrorInvalidDisplayFormat</fullName>
+        <categories>AutoNumber</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Invalid Display Format error message.</shortDescription>
+        <value>Invalid Display Format. See examples in "Learn more about display formats" section.</value>
```

Comment:
```suggestion
        <value>Invalid Display Format.</value>
```

Reply:
Because this is a field level error, I'm wondering if adding the second sentence about where to see examples might be overkill. But I defer to @gjang-sfdo.

Reply:
Cool!  I agree and am going to commit this, but let me know if you have other thoughts about this Grace!